sw/host/tests/chip/sysrst_ctrl/sysrst_ctrl_ec_rst_l.rs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::{bail, Context, ensure, Result};
+use anyhow::{bail, ensure, Context, Result};
```

Comment:
If you want to correct the formatting, maybe make that part of a different commit in the PR?

Reply:
ah, a  rustfmt_fix escape, how did the code make it into the tree despite presubmit checks?

I reverted unrelated files.

Reply:
The rustfmt_check is know to be flawed because we need to manually point to targets to verify. There is a pending PR to fix it but I requires an upstream change to rules_rust that I haven't gotten around to submit.